autoreject/conftest.py

```diff
@@ -28,6 +28,7 @@ def pytest_configure(config):
     ignore:FigureCanvasAgg is non-interactive.*:UserWarning
     ignore:`np.MachAr` is deprecated.*:DeprecationWarning
     ignore:`product` is deprecated as of NumPy.*:DeprecationWarning
+    ignore:datetime\.datetime\.utcfromtimestamp.* is deprecated and.*:DeprecationWarning
```

Comment:
@sappelhoff it turned out we were already using `pytest_configure` to set the ignore lines, so setting them in `setup.cfg` (as well) did not have the desired effect.

Reply:
ohhh thanks @larsoner, I should have checked that!